taxonomies/ingredients.txt

```diff
@@ -49982,6 +50234,7 @@ en:Mung bean sprout, bean sprouts, beansprouts
 da:bønnespirer
 de:Mungobohnensprosse, Mungobohnensprossen
 es:brotes de judia mungo
+fr:Germes de soja/Pousses de soja
```

Comment:
```suggestion
```
`fr:pousses de soja` already exists, and could possibly be merged with `en:sprouted soybeans`